opentelemetry-exporter-geneva/geneva-uploader-ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.2.0"
 edition = "2021"
 homepage = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-exporter-geneva/geneva-uploader-ffi"
 repository = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-exporter-geneva/geneva-uploader-ffi"
-rust-version = "1.75.0"
+rust-version = "1.85.0"
```

Comment:
I believe this is un-avoidable, as rust-sdk-for-azure needs this minimal version -  https://github.com/Azure/azure-sdk-for-rust/blob/bbb303181bcd3492f2595777ab822dc889adc493/Cargo.toml#L38?